Source/WebCore/platform/graphics/avfoundation/objc/LocalSampleBufferDisplayLayer.mm

```diff
@@ -182,16 +180,19 @@ static void runWithoutAnimations(const WTF::Function<void()>& function)
     m_rootLayer = adoptNS([[CALayer alloc] init]);
     m_rootLayer.get().hidden = hideRootLayer;
 
-    m_rootLayer.get().bounds = CGRectMake(0, 0, size.width(), size.height());
+    auto bounds = CGRectMake(0, 0, size.width(), size.height());
+    m_rootLayer.get().bounds = bounds;
+    m_rootLayer.get().position = { bounds.size.width / 2, bounds.size.height / 2 };
```

Comment:
The defaults for `anchorPoint` will be different on different platforms, so it may be worthwhile to set these to `{ 0.5, 0.5 }` here.  Also keep in mind that `position` is relative to the layer's parent's bounds. Shouldn't be a problem here if the m_rootLayer's parent always has the same bounds as it does.

Reply:
And same for m_sabpleBufferDisplayLayer, whose `anchorPoint` initialization was removed above.

Reply:
I will add back anchorPoint to m_sampleBufferDisplayLayer. Are you suggesting I do the same for the root layer even though we are not doing any transform to that layer?
Also which platform is not using `{ 0.5, 0.5 }`?